### ISSUE TITLE
Unsubscribe from notifications

### ISSUE THREAD

**Issue by FelixFleckensteinMentopolis** [openBackhaul/NotificationProxy#27]
Currently there is no obvious way for subscribers to any of the controller/devices-endpoints (like "notify-controller-attribute-value-changes") to unsubscribe again and will continue receiving messages until all controllers are removed via the "remove-controller"-endpoint. In BasicServices there exists the "end-subscription"-endpoint, should this be used to unsubscribe subscribers from specific subscriptions of IndividualServices in NP?

**Comment by FelixFleckensteinMentopolis** [comment]
We now have seen in config file, that "end-subscription" in BasicServices is the correct function.
If this understanding is correct, please close the question.

**Comment by openBackhaul** [comment]
Yes, I confirm that /v1/end-subscription shall be used for ending both basic and individual subscriptions.
If there would be no further need for clarification, please close the issue.
